src/main/java/com/example/TaskFlow/services/UserServiceImpl.java

```diff
@@ -28,7 +28,7 @@ public UserDTO createUser(UserDTO userDTO) {
         User userEntitySave = objectMapper.convertValue(userDTO, User.class);
         userValidationService.firstNameValidation(userDTO.getFirstName());
```

Comment:
Why do we have several validation methods for each user property? Can't we just validate the entire UserDTO object at once, including first name, last name, and email? We can have a more generic `validateUser` method, and inside that method, we can have other methods for validation: one for first and last name fields, one for the age, another for email, and so on. 
Also, please have a look on how we can validate the fields of a DTO. Have a look [here](https://medium.com/paysafe-bulgaria/springboot-dto-validation-good-practices-and-breakdown-fee69277b3b0).